.gitignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
I think we should still have the ESLint / TS files, because they may get created by these tools

```
.eslintcache
*.tsbuildinfo
```

Reply:
958e912